apps/hosts/admin.py

```diff
@@ -1,3 +1,20 @@
 from django.contrib import admin
 
-# Register your models here.
+from base.admin import TimeStampedAdmin
+
+from .models import (ChallengeHostAdmin,
+                     ChallengeHostTeamAdmin)
+
+
+@admin.register(ChallengeHostTeamAdmin)
+class HostTeamModelAdmin(TimeStampedAdmin):
+    list_display = ("team_name", "created_by")
+    list_filter = ("team_name", "created_by")
+    search_fields = ("team_name", "created_by")
+
+
+@admin.register(ChallengeHostAdmin)
+class HostModelAdmin(TimeStampedAdmin):
+    list_display = ("user", "team_name", "status")
+    list_filter = ("user", "team_name", "status")
```

Comment:
lets give search by `user` and `team_name` and filter by `status` and `permissios`. can you please do these changes?